src/glossary.rst

```diff
@@ -445,13 +445,15 @@ Glossary
    suite log
    suite log directory
       A Cylc suite logs events and other information to the suite log files
-      when it runs. There are three log files:
+      when it runs. There are four log files:
 
       * ``out`` - the stdout of the suite.
       * ``err`` - the stderr of the suite, which may contain useful debugging
         information in the event of any error(s).
       * ``log`` - a log of suite events, consisting of information about
         user interaction.
+      * ``file-installation-log`` - a log documenting the file installation 
```

Comment:
I have used the terminology `remote platform` throughout as I believe the plan is for users to remain unaware of the `install target` term.